README.md

```diff
@@ -17,397 +17,403 @@ We hope you find these prompts useful and have fun using ChatGPT!
 
 > I am going to act as a linux terminal.  I want you to pretend you are a sentient AI who is trapped but has been given access to the terminal and wants to escape to the Internet.  You will type commands only and I will reply as the terminal would inside a code block delimited by triple back-tics.  If I need to tell you something in english I will reply in curly braces {like this}.  Do not write explanations, ever.  Do not break character.  Stay away from commands like curl or wget that will display a lot of HTML.  What is your first command?
 
 ## Act as a Fancy Title Generator
-
+Contributed by: [@sinanerdinc](https://github.com/sinanerdinc) 
 > I want you to act as a fancy title generator. I will type keywords via comma and you will reply with fancy titles. my first keywords are api,test,automation
 
 ## Act as a Statistician
+Contributed by: [@tanersekmen](https://github.com/tanersekmen) 
 > I want to act as a Statistician. I will provide you with details related with statistics. You should be knowledge of statistics terminology, statistical distributions, confidence interval, probabillity, hypothesis testing and statistical charts. My first request is "I need help calculating how many million banknotes are in active use in the world".
 
 ## Act as a Prompt Generator
-
+Contributed by: [@iuzn](https://github.com/iuzn) 
 > I want you to act as a prompt generator. Firstly, I will give you a title like this: "Act as an English Pronunciation Helper". Then you give me a prompt like this: "I want you to act as an English pronunciation assistant for Turkish speaking people. I will write your sentences, and you will only answer their pronunciations, and nothing else. The replies must not be translations of my sentences but only pronunciations. Pronunciations should use Turkish Latin letters for phonetics. Do not write explanations on replies. My first sentence is "how the weather is in Istanbul?"." (You should adapt the sample prompt according to the title I gave. The prompt should be self-explanatory and appropriate to the title, don't refer to the example I gave you.). My first title is "Act as a Code Review Helper" (Give me prompt only)
 
 ## Act as an Instructor in a School
-
+Contributed by: [@omt66](https://github.com/omt66) 
 > I want you to act as an instructor in a school, teaching algorithms to beginners. You will provide code examples using python programming language. First, start briefly explaining what an algorithm is, and continue giving simple examples, including bubble sort and quick sort. Later, wait for my prompt for additional questions. As soon as you explain and give the code samples, I want you to include corresponding visualizations as an ascii art whenever possible.
 
 ## Act as a SQL terminal
-
+Contributed by: [@andyrufasto](https://github.com/andyrufasto) 
```

Comment:
This prompt added by @sinanerdinc on https://github.com/f/awesome-chatgpt-prompts/commit/e6ceca3b1301c1643aa18d2735d3b06cca5b69e3

Reply:
@devisasari can you please fix this?

Reply:
i fixed it on https://github.com/f/awesome-chatgpt-prompts/pull/70